apps/services/sessions/src/app/sessions/sessions-query.dto.ts

```diff
@@ -18,6 +18,15 @@ export class SessionsQueryDto extends PaginationDto {
   })
   to?: Date
 
+  @IsOptional()
+  @IsString()
+  @Length(2, 2)
+  @ApiPropertyOptional({
+    description:
+      'Only return sessions originating from this country code (see: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).',
+  })
+  ipLocation?: string
```

Comment:
Could you consolidate the attribute with the same attribute in `create-session.dto.ts`?

Reply:
what do you mean?

Reply:
There's a near-identical attribute with the same name in `create-session.dto.ts`; I'm wondering whether they can be united to reduce boilerplate and duplicate code.

Reply:
This is the query class, so it would require sublassing and in my opinion not worth the refactor.